cvat/apps/annotation/ddln_spotter.py

```diff
@@ -20,6 +20,23 @@
     ],
 }
 
+YML_TEMPLATE = """sources:
+  - {ddln_id}
+date: {curr_date}
+team: {team}
```

Comment:
this is not valid YAML
the task is filed incorrectly, fix the format to produce valid YAML

Reply:
fixed!